skills.js

```diff
@@ -1,54 +1,85 @@
 'use strict';
 const constants = require('./constants');
 
+// Effect functions for each skill
+
+//TODO: Real calculations for effects
+var useSlice = function (user, target) {
+    target.hp += -5 + (user.strength / 2);
+}
+
+var useSwipe = function (user, target) {
+    target.hp += -3 + (user.strength / 2);
+}
+
+var useSmallHeal = function (user, target) {
+    target.hp += 3 + (user.strength / 2);
+}
+
+var useBigHeal = function (user, target) {
+    target.hp += 8 + (user.strength / 2);
+}
+
+var useCounterAttack = function (user, target) {
+    target.hp -= 4 + (user.strength / 2);
+}
+
+var useArmour = function (user, target) {
+    target.endurance += 1 + (user.strength / 2);
+}
+
+var useDrain = function (user, target) {
+    target.strength -= 1 + (user.strength / 2);
+}
+
 module.exports = [
-  {
+{
     name: 'slice',
     type: constants.ATTACKING,
```

Comment:
Probably better to keep them for now. Although they're not used in battle, they still define what the general purpose of the skill is.